lang/cs/Org.Apache.REEF.Examples.HelloREEF/Readme.md

```diff
@@ -6,4 +6,25 @@ This project contains a simple example of a REEF Program. It contains the follow
   * `HelloTask`: This Task prints a greeting to STDOUT of the Evaluator.
 
 ## Running it
-Just run the main class, `HelloREEF`, followed by the runtime you want, e.g. `local`.
\ No newline at end of file
+Just run the main class, `HelloREEF`, followed by the runtime you want, e.g. `local`.
```

Comment:
I am trying to add NuGet dependency to reduce the manual steps. REEF-1841. You can try that PR to see if it can resolve the issue.